src/main/java/seedu/address/logic/commands/RestockIngredientCommand.java

```diff
@@ -65,11 +63,8 @@ public CommandResult execute(Model model, CommandHistory history) throws Command
         Ingredient ingredientToRestock = lastShownList.get(index.getZeroBased());
         Ingredient restockedIngredient = createRestockedIngredient(ingredientToRestock, quantityToRestock);
 
-        try {
-            model.setIngredient(ingredientToRestock, restockedIngredient);
-        } catch (DuplicateItemException e) {
-            throw new CommandException(MESSAGE_DUPLICATE);
-        }
+        model.setIngredient(ingredientToRestock, restockedIngredient);
```

Comment:
createRestockedIngredient doesn't check for integer overflow. Adding 1000000000 a few times causes it to go negative